irohad/ametsuchi/impl/storage_impl.hpp

```diff
@@ -44,6 +45,10 @@ namespace iroha {
 
     class StorageImpl : public Storage {
      protected:
+      static void createDatabaseIfNotExist(
```

Comment:
I think this function has to return a result with a status of the operation. Such as:
```
Result<bool, exeption>
``` 
or something like this.

Reply:
Might be Result<void, message_type>

Reply:
True for the new one and false for existed db.